flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -681,20 +677,16 @@ public void testManagedMemoryFractionForSpecifiedResourceSpec() throws Exception
 
 		final StreamConfig sourceConfig = new StreamConfig(vertex1.getConfiguration());
 		assertEquals(0.1, sourceConfig.getManagedMemoryFractionOnHeap(), 0.000001);
-		assertEquals(0.4, sourceConfig.getManagedMemoryFractionOffHeap(), 0.000001);
```

Comment:
Not really. We need this change because we have change the setting up part of this test cases, from setting both on-heap/off-heap managed memory in `ResourceSpec` to managed memory only. If we do not change the validation part, the test case will fail.

Reply:
But I think I should remove validation for on-heap and keep it for off-heap. The setting up part as well.

Reply:
alright, makes sense